content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Aplicando os Padrões de Segurança do Pod Através da Configuração do Controlador de Admissão Embutido
+content_type: task
+min-kubernetes-server-version: v1.22
+---
+
+Desde a versão v1.22, o Kubernetes fornece um [controlador de admissão](/docs/reference/access-authn-authz/admission-controllers/#podsecurity)
+embutido para fazer cumprir os [padrões de segurança do Pod](/docs/concepts/security/pod-security-standards).
+Você pode configurar esse controlador de admissão para definir padrões em todo 
+o cluster e [exceções](/docs/concepts/security/pod-security-admission/#exemptions).
+
+## {{% heading "prerequisites" %}}
+
+{{% version-check %}}
+
+- Garanta que a `PodSecurity` do [`feature gate`](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features) 
```

Comment:
"`PodSecurity` do feature gate" is reading a bit weird to me. What we want the reader to make sure is that the feature gate named `PodSecurity` is enabled. Maybe "o _feature gate_ `PodSecurity` está ativado"?

It's also better to format _feature gate_ as italic rather than code formatting.